src/libexec/zmsearchndelete

```diff
@@ -0,0 +1,946 @@
+#!/usr/bin/perl
+#
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2009, 2018 Synacor, Inc.
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+
+# TBD: need these options/capabilities?
+#    --proc <procnum>       number of child processes allowed (4)
+
+=head1 NAME
+
+zmsearchndelete - Search messages with specified string and delete it
+
+=head1 SYNOPSIS
+
+zmsearchndelete [options] [<userlistfile> ...]
+
+  Options: [*] == Required, [+] == Multiple vals OK, (val) == Default
+    --url <url>            [*]URL of soap service
+    --authuser <user>      [*]account for authentication
+    --password <password>  [*]password for authuser
+    --query 		   [*]search string specified in "" boundary
```

Comment:
nit: there are a few tabs embedded in this file (this line being the first) - please replace all tabs with spaces and align properly

Reply:
done